gspread/spreadsheet.py

```diff
@@ -743,7 +745,11 @@ def list_protected_ranges(self, sheetid):
 
         return sheet.get("protectedRanges", [])
 
-    def refresh_lastUpdateTime(self):
-        """Refresh the lastUpdateTime property of the spreadsheet."""
-        metadata = self.client._get_file_drive_metadata(self.id)
+    def get_lastUpdateTime(self):
+        """Get the lastUpdateTime metadata from the Drive API.
+        Also updates the cached value in the _properties dict.
+        """
+        metadata = self.client.get_file_drive_metadata(self.id)
+        # remove this upon deprecation of lastUpdateTime @property
         self._properties["modifiedTime"] = metadata["modifiedTime"]
+        return metadata["modifiedTime"]
```

Comment:
if this is just a matter of API, may be we can keep both options os users already using the `refresh_lastUpdateTime` will keep using it as it is and can keep using the property. Users interedted in the new method can use it too along side the refresh ethod.

this way I suppose we could be able to merge it in v5 right ? :thinking: